sql/errors.go

```diff
@@ -588,6 +588,12 @@ var (
 
 	// ErrInvalidTextIndex is returned for an index on a byte column with no prefix or an invalid prefix
 	ErrInvalidTextIndex = errors.NewKind("index on text column '%s' unsupported")
+
+	// ErrInvalidTableEntry is returned when a table gets a key that can't be converted to an Entry
+	ErrInvalidTableEntry = errors.NewKind("The %s key for the `%s` table was given an unknown entry")
```

Comment:
This is a bad error message. What is it trying to convey? Why doesn't it echo the value that was given?